session/pingpong/invoice_tracker.go

```diff
@@ -288,18 +284,12 @@ func (it *InvoiceTracker) revealPromise() error {
 		return nil
 	}
 
-	err = it.deps.AccountantCaller.RevealR(accountantPromise.R, it.deps.ProviderID.Address, accountantPromise.AgreementID)
-	if err != nil {
-		log.Error().Err(err).Msg("Could not reveal R")
-		it.incrementAccountantFailureCount()
-		if it.getAccountantFailureCount() > it.deps.MaxAccountantFailureCount {
-			return errors.Wrap(err, "max failure count calling accountant reached")
-		}
-		log.Warn().Msg("Ignoring accountant error, we haven't reached the error threshold yet")
-		return errHandled
+	err = it.deps.AccountantCaller.RevealR("wazap", it.deps.ProviderID.Address, accountantPromise.AgreementID)
```

Comment:
Why it started to be `wazap` instead of `accountantPromise.R`? Is it ok to have here a const string?

Reply:
leftovers from testing, fixed